test-network-function/accesscontrol/suite.go

```diff
@@ -46,132 +44,104 @@ var _ = ginkgo.Describe(common.AccessControlTestKey, func() {
 		ginkgo.BeforeEach(func() {
 			common.ReloadConfiguration(&configData)
 		})
-
+		common.ReloadConfiguration(&configData)
```

Comment:
@jun-rh  is actually rewriting this part to create a singleton with new name.
you guys should figure out which one should go first, the other will have to fix some conflict :)

Reply:
done the merge